package.json

```diff
@@ -148,9 +146,9 @@
     "webpack": "^5.64.3"
   },
   "engines": {
-    "node": "12"
+    "node": "10.* || >= 12"
```

Comment:
Hoewel dit naar ik begreep niets echt afdwingt, denk ik dat dit niet meer correct is. Zie https://github.com/embroider-build/embroider/releases/tag/v0.42.0

Reply:
Dwingt inderdaad niets af tenzij je `engine-strict=true` zou toevoegen aan `.npmrc`. Maken we er dan `>= 16` van? Dat is ook de node version in `madnificent/ember:3.28.5`.